graph/src/env/mod.rs

```diff
@@ -391,6 +401,11 @@ struct Inner {
     enable_gas_metrics: EnvVarBoolean,
     #[envconfig(from = "GRAPH_EXPERIMENTAL_TIMESERIES")]
     enable_timeseries: Option<bool>,
+    #[envconfig(from = "GRAPH_HISTORY_BLOCKS_OVERRIDE")]
+    history_blocks_override: Option<BlockNumber>,
+    // TODO: Find a good default value
+    #[envconfig(from = "GRAPH_MIN_HISTORY_BLOCKS", default = "10000")]
+    min_history_blocks: BlockNumber,
```

Comment:
No default here, and an empty value means never override.

Reply:
I dont understand> No default for GRAPH_MIN_HISTORY_BLOCKS ?

Reply:
Ah I confused this with `history_blocks_override`. Sorry, ignore this comment.